pkg/internal/snowflakeenvs/snowflake_environment_variables.go

```diff
@@ -0,0 +1,10 @@
+package snowflakeenvs
```

Comment:
should this not be under acceptance package?

Reply:
acceptance package is used for tests only whereas this one can be used for production code too.